### DISCUSSION TITLE
Suggestion: Consolidate the posts of someone you follow (daily, weekly, etc.)

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#23263] [General]
There are some accounts that post 25x+ per day which ends up flooding timelines. While muting is a great option, there are some cases where you do want to read that account's posts, just not sprawled out, burying the posts of those who are not as prolific. 

It would be great to have an option to set posts from specific accounts to collapse under one entry set to post at specific intervals (hourly, daily, weekly, etc.).

**Comment by trwnh** [comment]
#2089
